tools/rpm2img

```diff
@@ -152,10 +152,15 @@ mv "${ROOT_MOUNT}/boot"/* "${BOOT_MOUNT}"
 
 # Set the Bottlerocket variant, version, and build-id
 SYS_ROOT="${ARCH}-bottlerocket-linux-gnu/sys-root"
-echo "PRETTY_NAME=\"${PRETTY_NAME} ${VERSION_ID}\"" >> "${ROOT_MOUNT}/${SYS_ROOT}/usr/lib/os-release"
+VERSION="${VERSION_ID} (${VARIANT})"
+echo "VERSION=\"${VERSION}\"" >> "${ROOT_MOUNT}/${SYS_ROOT}/usr/lib/os-release"
```

Comment:
nit: probably time for a `cat <<EOF >>${ROOT_MOUNT}/${SYS_ROOT}/usr/lib/os-release` rather than repeating echo/filename every time.